lang/squeak/Portfile

```diff
@@ -122,3 +114,5 @@ variant x11 {
         configure.args-delete   --without-x
         configure.args-append   --with-x
 }
+
+default_variants                +x11
```

Comment:
You must only default to +x11 if +quartz has not already been selected. See https://trac.macports.org/wiki/PortfileRecipes#default_variants or any other port that has +x11 and +quartz variants.

Reply:
Indeed, thank you